diff --git a/dist/index.mjs b/dist/index.mjs
index 92a80377692488c4ba8801ce33e7736ad7055e43..add6281bbecaa1c03d3b48eb99aead4a7a7336b2 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -1593,7 +1593,7 @@ function prepareCallSettings({
   return {
     maxTokens,
     // TODO v5 remove default 0 for temperature
-    temperature: temperature != null ? temperature : 0,
+    temperature: temperature,
     topP,
     topK,
     presencePenalty,
